errors/errors.js

```diff
@@ -0,0 +1,91 @@
+const JSONAPIError = require('jsonapi-serializer').Error;
+const _ = require('lodash');
+
+/**
+ * @summary Construct error object
+ * @function
+ * @param {string} status The HTTP status code
+ * @param {string} title A short, human-readable summary of the problem
+ * @param {string} code An application-specific error code
+ * @param {string} detail A human-readable explanation
+ * @returns {Object} An error object
+ */
+const error = (status, title, code, detail) => ({
+  status,
+  title,
+  code,
+  detail,
+  links: { about: `https://developer.oregonstate.edu/documentation/error-reference#${code}` },
+});
+
+/**
+ * @summary [400] Return a Bad Request error object
+ * @function
+ * @param {[string]} details An array of bad request details
+ * @returns {Object} Bad Rquest error object
+ */
+const badRequest = (details) => {
+  const badRequests = [];
+  _.forEach(details, detail => badRequests.push(error('400', 'Bad Request', '1400', detail)));
+  return new JSONAPIError(badRequests);
+};
+
+/**
+ * @summary [401] Return a Unauthorized error object
+ * @function
+ * @returns {Object} Unauthorized error object
+ */
+const unauthorized = () => new JSONAPIError(error('401', 'Unauthorized', 'Unauthorized'));
```

Comment:
This call is setting the error code to "Unauthorized", I think it should be 1401 and detail should be "Unauthorized" or some further detailed explanation to the error. Possibly "credentials are invalid" or something.